posthog/hogql_queries/error_tracking_query_runner.py

```diff
@@ -27,15 +29,89 @@ def __init__(self, *args, **kwargs):
 
     def to_query(self) -> ast.SelectQuery:
         return ast.SelectQuery(
-            select=[parse_expr(x) for x in self.query.select],
+            select=self.select(),
             select_from=ast.JoinExpr(table=ast.Field(chain=["events"])),
-            where=self._where(),
-            order_by=self._order_by(),
-            group_by=[ast.Field(chain=["events", "properties", "$exception_fingerprint"])],
+            where=self.where(),
+            order_by=self.order_by,
+            group_by=self.group_by(),
+        )
+
+    def select(self):
+        exprs: list[ast.Expr] = [
+            ast.Alias(alias="occurrences", expr=ast.Call(name="count", args=[])),
+            ast.Alias(
+                alias="sessions", expr=ast.Call(name="count", distinct=True, args=[ast.Field(chain=["$session_id"])])
+            ),
+            ast.Alias(alias="last_seen", expr=ast.Call(name="max", args=[ast.Field(chain=["timestamp"])])),
+            ast.Alias(alias="first_seen", expr=ast.Call(name="min", args=[ast.Field(chain=["timestamp"])])),
+            ast.Alias(alias="error", expr=ast.Call(name="any", args=[ast.Field(chain=["properties"])])),
+        ]
+
+        # "any(properties) as error",
+
+        if self.query.fingerprint:
+            # Include the event data when looking at an individual fingerprint
+            exprs.append(
+                ast.Alias(
+                    alias="events",
+                    expr=ast.Call(
+                        name="groupArray",
+                        args=[
+                            ast.Call(
+                                name="tuple",
+                                args=[
+                                    ast.Field(chain=["uuid"]),
+                                    ast.Field(chain=["properties"]),
+                                    ast.Field(chain=["timestamp"]),
```

Comment:
Planning on making the query more generic so that you can add `eventColumns` rather than having the definite list here